### ISSUE TITLE
Microsoft Edge is not being installed

### ISSUE THREAD

**Issue by XenoPanther** [gus33000/UUPMediaCreator#231]
I have just finished installing 27695 and noticed that Edge wasn't installed. I can manually install it, but I just found this weird

**Comment by gus33000** [comment]
Yeah starting with some builds edge is now a separate file/components and we dont handle it yet. Though in the past it also was still present in the image but it appears now it is not.
